library/std/src/Std/BlochSphere.qs

```diff
@@ -0,0 +1,95 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/// # Summary
+/// Maps a Pauli axis from one direction to another by applying an appropriate Clifford transformation.
+/// For example, use `within MapPauliAxis(PauliZ, PauliX, q)` and perform Z rotation when X rotation is desired.
+///
+/// # Description
+/// This function applies single-qubit Clifford transformations that remap Pauli axes on a Bloch sphere.
+/// These mappings are useful to apply operations known for one Pauli basis in different Pauli bases.
+/// Provide `from` and `to` parameters in terms of a passive transformation. For example,
+/// when a rotation around the X axis is desired and a rotation around the Z axis is available,
+/// the Z axis gets pointed in the direction of the X axis so that the rotation around Z axis can be used.
+/// Out of several transformations that achieve the requested mapping, the one with fewer gates is used.
+///
+/// # Remarks
```

Comment:
In our other doc comments, the `# Remarks` section goes after the `# Input` section, I believe.